libmproxy/console/__init__.py

```diff
@@ -496,11 +496,13 @@ def toggle_eventlog(self):
         self.eventlog = not self.eventlog
         self.view_flowlist()
 
-    def _readflow(self, path):
-        path = os.path.expanduser(path)
+    def _readflow(self, paths):
         try:
-            f = file(path, "rb")
-            flows = list(flow.FlowReader(f).stream())
+            flows = []
+            for path in paths:
+                path = os.path.expanduser(path)
+                f = file(path, "rb")
```

Comment:
Could you adjust this to `with open(path, "rb"):` here and below? We should make sure that we close all files.